src/main/java/com/xebialabs/deployit/ci/Credential.java

```diff
@@ -116,6 +113,7 @@ public boolean isUseGlobalCredential() {
 
     public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Project context) {
         // TODO: also add requirement on host derived from URL ?
+        Jenkins.getInstance().checkPermission(Item.CONFIGURE);
```

Comment:
Not a useful permission to check on the root object. If this is called as part of "Configure System" only, use 
```suggestion
        Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);
```

Otherwise https://github.com/jenkinsci/credentials-plugin/blob/master/docs/consumer.adoc has the canonical doc for what permission to check based on `@AncestorInPath`.